shared/actions/tracker.js

```diff
@@ -593,91 +449,86 @@ function _serverCallMap(
         // This is the first spot that we have access to the user, so let's use that to get
         // The user information
 
-        dispatch({
-          type: Constants.setProofs,
-          payload: {username, identity},
-        })
-        dispatch({type: Constants.updateProofState, payload: {username}})
+        dispatch(TrackerGen.createSetProofs({username, identity}))
+        dispatch(TrackerGen.createUpdateProofState({username}))
         if (identity.breaksTracking && !isGetProfile) {
-          dispatch({type: Constants.showTracker, payload: {username}})
+          dispatch(TrackerGen.createShowTracker({username}))
         }
       })
     },
-    'keybase.1.identifyUi.displayTrackStatement': (params, response) => {
+    'keybase.1.identifyUi.displayTrackStatement': (_, response) => {
       response.result()
     },
 
-    'keybase.1.identifyUi.dismiss': ({username, reason}, response) => {
+    'keybase.1.identifyUi.dismiss': (_, response) => {
```

Comment:
`{username, reason}` is replaced with `_` here, but you still access username below...

Reply:
yeah how this works is a bit crappy. the username should be static through this entire flow so it works cause its defined above. in practice it should always be the same. anyways i put back username